src/whole_body_state_conversions/whole_body_interface.py

```diff
@@ -123,6 +123,8 @@ def writeToMessage(self, t, q, v=None, tau=None, p=dict(), pd=dict(), f=dict(),
                 contact_msg.wrench.torque.x = force.angular[0]
                 contact_msg.wrench.torque.y = force.angular[1]
                 contact_msg.wrench.torque.z = force.angular[2]
+                if np.linalg.norm(force.linear) > 0.:
```

Comment:
```suggestion
                if len(contact_info) == 3:
                    contact_msg.status = contact_info[2]
```

Reply:
Done! I have added a deprecate message to keep the previous behaviour.

Note that I have a similar code in whole-body-state-msgs. And there is legacy code that uses.
I will remove the deprecation routine as soon as I update my code.